doc/developer_doc/41-setup-cvmfs-gateway.md

```diff
@@ -0,0 +1,166 @@
+# Setup of CernVM-FS Gateway
+
+The idea behind the gateway is that only the gateway has access to CernVM-FS storage and administrates transactions via leases.
+This allows multiple publishers to modify data in the same repository but at different data locations within the repository.
+
+**Note**: At the moment, the gateway does not support container publishing via *publishers*.
+
+## Setup Gateway
+
+Setting up a CernVM-FS gateway consists of 3 steps:
+
+1. Create repository 
+2. Create Gateway key that publisher will use for authentication
+3. Make repository accessible via Gateway
+
+
+### Example: Create gateway for repository `test.gw.repo`
+
+(Note: every command is done in `sudo`)
+
+1. **Create repository**
+
+```py
+  # General setup
+  cvmfs_config setup
```

Comment:
This is actually not needed if you _only_ run a server/gateway.  If you also want to mount the regular client though, then you'd need to run the `cvmfs_config setup` command.